### DISCUSSION TITLE
Issue with Artisan Sliders

### DISCUSSION THREAD

**Discussion by Turkis306** [artisan-roaster-scope/artisan#991] [Q&A]
I am working with a Coffed SR3 delta+ roaster and I am having an issue with running the roaster on Artisan , when I start artisan the sliders read the BT and ET temperatures and I am not sure if the problem is with the software or the roaster. I tried reinstalling the software and downloading the latest version 2.8, but even when I am warming up the roasters the slider keep moving to the temperature number.

**Comment by MAKOMO** [comment]
This is most likely a configuration issue. You might have some event quantifiers enabled that observe BT or ET. Just deactivate them under Config >> Events, 4th tab "Quantifier" by unchecking the first flag in each of the 4 rows.
